Fabric.Authorization.AccessControl/src/app/modules/access-control/custom-group/custom-group.component.ts

```diff
@@ -363,6 +363,16 @@ export class CustomGroupComponent implements OnInit, OnDestroy {
                   .then(value => {
                       return this.edwAdminService.syncGroupWithEdwAdmin(newGroup.groupName)
                           .toPromise().then(o => { return value; }).catch(err => { return value; });
+                  })
+                  .then(value => {
+                      if (usersToRemove) {
+                        let response;
+                        usersToRemove.forEach(element => {
+                          response = this.edwAdminService.syncUserWithEdwAdmin(element.subjectId, element.identityProvider)
```

Comment:
this might get a little chatty if we are removing a large list of users...can we add an endpoint that takes a list of users instead?

Reply:
ah, we can look over that for sure.   Maybe replace our single user and make it take it a list regardless.